src/components/TabbedNav/TabbedNav.jsx

```diff
@@ -0,0 +1,42 @@
+const React = require('react')
+const hookUpStickyNav = require('../hookUpStickyNav')
+
+class TabbedNav extends React.Component {
+  render() {
+    return (
+      <nav id="tabbed-nav" className="top-nav nav tabs is-toggle is-fullwidth">
+        <ul>
+          <li className={this.props.active === 'tasks' ? 'is-active' : ''}>
+            <a onClick={this.props.showTasks}>
```

Comment:
Look again, your button is now `is-link`! 💎  See http://bulma.io/documentation/elements/button/ for the `is-link` class Bulma provides for buttons. I think using `button is-link` together on a button here would make it look basically right.